doc/cmake.txt

```diff
@@ -209,6 +210,32 @@ cq			Close the CMake console window.
 
 <C-C>			Stop the current command.
 
+==============================================================================
+Events							*cmake-events*
+
+To customize behaviour after a command has finished, Vim-CMake calls user
+autocommands
+
+List of events~
+
+CMakeBuildFailed
+CMakeBuildSucceeded
+
+Example usage of CMakeBuildFailed to jump to the first error
+>
+      let g:jump_on_error = 0 " We do not want to focus the console
+      augroup vim-cmake-group
+	autocmd User CMakeBuildFailed :cfirst
+      augroup END
+<
+
+Example usage of CMakeBuildSucceeded to close the Vim-CMake console
+>
+      augroup vim-cmake-group
+	autocmd User CMakeBuildSucceeded call cmake#console#Close()
```

Comment:
cmake#console#Close() is not part of the user Vim-CMake API, please replace "cmake#console#Close()" with "CMakeClose"